src/main/java/org/prebid/server/proto/openrtb/ext/request/rubicon/ExtImpRubicon.java

```diff
@@ -32,4 +32,7 @@
     RubiconVideoParams video;
 
     String pchain;
+
+    @JsonProperty("is_rewarded_inventory")
```

Comment:
How the changes to Rubicon bidder are related to adding a new bidder? Please roll back this change and make a separate PR if this is not mistake and changes are required.